packages/dara-components/js/common/input/input.tsx

```diff
@@ -24,6 +24,9 @@ interface InputProps extends FormComponentProps {
 const StyledInput = injectCss(UiInput);
 const StyledNumericInput = injectCss(UiNumericInput);
 
+function getNumericValue(value: string | number): number {
+    return value == null || Number.isNaN(+value) ? null : +value;
```

Comment:
What's the purpose of using `+` rather than `Number()` here?

Reply:
They both behave the same way here, I learned about it the other day and had just been looking for an excuse to try it.

Reply:
I would argue `Number` is more explicit, with `+` for example I had to think twice whether there wouldn't be some edge case with negative numbers.

Reply:
That is fair, I have switched to use `Number`